### ISSUE TITLE
Downloadfolder subdirectorys

### ISSUE THREAD

**Issue by Tockra** [pyload/pyload#3243]
Hi,

If you currently download something you can choose the directory where to store the downloaded files. 
But you can't create a subdirectory.

For example:
I want to download my files to 
.../downloads/NewFolder/NewFolder2

If I choose directoryname Newfolder/Newfolder2 the downloader just creates one directory.

T

**Comment by Velociraptor85** [comment]
@Tockra Which OS is this on? 
can you specify just Newfolder does this work? and after that Newfolder2?

**Comment by Tockra** [comment]
![unbenannt](https://user-images.githubusercontent.com/1353859/47112739-7fa04e00-d257-11e8-80c0-7b7581673a39.png)

**Comment by GammaC0de** [comment]
This the package name, it is not download folder.
pyload does not support this.

**Comment by Tockra** [comment]
Yes I know, but it would be nice to add there the feature to create folders (if you set the option to create folder for packages).
